packages/uma-auth-client/src/NwcRequester.ts

```diff
@@ -18,11 +18,12 @@ export interface NwcConnection {
 }
 
 export class NwcRequester {
-  relay: Relay;
-  relayUrl: string;
+  relay!: Relay;
+  relayUrl!: string;
+  walletPubkey!: string;
```

Comment:
Not sure how to avoid the `!` since I want to reuse the initNwcConnection logic in the constructor when refreshing a token and ts is complaining these aren't being initialized in the constructor 🤔